sdcm/cluster_aws.py

```diff
@@ -698,7 +698,7 @@ def restart(self):
                 self.remoter.sudo(shell_script_cmd(f"""\
                     sed -e '/.*scylla/s/^/#/g' -i /etc/fstab
                     sed -e '/auto_bootstrap:.*/s/false/true/g' -i /etc/scylla/scylla.yaml
-                    if ! grep replace_address_first_boot: /etc/scylla/scylla.yaml; then
+                    if ! grep ^replace_address_first_boot: /etc/scylla/scylla.yaml; then
```

Comment:
So every next run we will add new line additional to existent?

Reply:
The `replace_address_first_boot: xxx` option will be commented in the end of restart().
I didn't remove it because we can know the restart times by the the commented options.

So the next run will add a new line, which is only valid.

Reply:
use `sdcm.cluster.BaseNode.remote_scylla_yaml` to make changes in the `scylla.yaml`

Reply:
@bentsi  This is not relevant to the quick fix we need here.